waspc/src/Wasp/Generator/FileDraft/CopyDirFileDraft.hs

```diff
@@ -0,0 +1,36 @@
+module Wasp.Generator.FileDraft.CopyDirFileDraft
+  ( CopyDirFileDraft (..),
+  )
+where
+
+import Control.Monad (when)
+import StrongPath
+  ( Abs,
+    Dir',
+    Path',
+    Rel,
+    (</>),
+  )
+import qualified StrongPath as SP
+import Wasp.Generator.Common (ProjectRootDir)
+import Wasp.Generator.FileDraft.Writeable
+import Wasp.Generator.FileDraft.WriteableMonad (WriteableMonad (copyDirectoryRecursive, createDirectoryIfMissing), doesDirectoryExist)
+
+-- | File draft based on another dir that is to be recursively copied.
```

Comment:
Maybe we could also mention here that this FileDraft is a bit unusual in the sense that it is Dir and not File, but due to Linux philosophy ... . So basically same what you wrote for FileDraft, but also here, to make people aware that this specific FileDraft is a bit unusual.